### ISSUE TITLE
Thunar

### ISSUE THREAD

**Issue by Bajoja** [Bajoja/indicator-kdeconnect#42]
I read this [article](http://docs.xfce.org/xfce/thunar/send-to), and it says is possible to add a script to "/usr/share//Thunar/sendto/", can someone here that use Thunar test this:

[kdeconnect-send-thunar.desktop.zip](https://github.com/Bajoja/indicator-kdeconnect/files/775684/kdeconnect-send-thunar.desktop.zip)

So if it work, let me know to add it on next release.

**Comment by tsjnachos117** [comment]
It seems to work for me.

I do have one question, however: is the last line of that file (specifically "MimeType=image/jpeg") necessary? I almost complained about the file not working (it wasn't showing up in the menu), until I noticed it's only for JPEGs. Just for fun, I commented that line out, and sent myself a random ZIP file without issues, so this file clearly works on zips as well.

The same cannot be said about directories, however. When I tried to send myself a directory,  I got the usual window asking me which device to send the directory to. After I selected my device, absolutely nothing happened. Likewise when I tried to send that same directory manually via the kdeconnect-send command, so I guess this has nothing to do with the .desktop file, itself.
